framework/docker/hyperlane/core_config.go

```diff
@@ -0,0 +1,42 @@
+package hyperlane
+
+// ProxyAdminCfg models the proxy admin address and owner fields.
+type ProxyAdminCfg struct {
+	Address string `yaml:"address"`
+	Owner   string `yaml:"owner"`
+}
+
+// HookCfg models a generic hook address + type.
+type HookCfg struct {
```

Comment:
Probably better to have concrete types for each, I assume this is `MerkleTreeHook`? 

I know the structure itself is very generic and could be reused...